junit5/pom.xml

```diff
@@ -16,14 +16,25 @@
     <properties>
 
         <version.junit.jupiter>5.8.2</version.junit.jupiter>
+        <version.junit.platform>1.8.1</version.junit.platform>
         <version.maven-compiler-plugin>3.3</version.maven-compiler-plugin>
 
         <!-- Azure SDK dependencies & other due to dependencyManagement issues with Azure SDK-->
-        <version.azure-resourcemanager>2.18.0</version.azure-resourcemanager>
-        <version.com.azure.bom>1.2.5</version.com.azure.bom>
+        <!-- Based on https://github.com/Azure/azure-sdk-for-java/tree/main/sdk/resourcemanager#include-the-package-->
+        <version.azure-resourcemanager>2.20.0</version.azure-resourcemanager>
+        <version.com.azure.azure-identity>1.6.1</version.com.azure.azure-identity>
+        <version.com.azure.azure-core-http-netty>1.12.6</version.com.azure.azure-core-http-netty>
+
+        <!-- Azure SDK uses okhttp3 which is also brought by Sunstone core...we need to find suitable version... -->
+        <version.com.squareup.okhttp3>4.10.0</version.com.squareup.okhttp3>
```

Comment:
Sunstone core and all "old" code uses okhttp and Sunstone JUnit5 module does not. It may not be worth the effort move legacy code to the new library. I would rather move away from Sunstone core ;)